src/assets/stylesheets/shared/_layout.scss

```diff
@@ -585,3 +585,35 @@ $wizard-footer-color: $color-pf-black-200;
 
 }
 
+span,div
+{
+  &.wizard-status-success{
+    color:lime;
```

Comment:
these should follow the PatternFly alert colors for success, warning, failure and info:

alert-success: pf-green-400
alert-info: pf-black-500
alert-warning: pf-orange-400
alert-danger: pf-red-100